Ryujinx.Ava/AppHost.cs

```diff
@@ -1046,7 +968,7 @@ private bool UpdateFrame()
                 }
             }
 
-            // Touchscreen
+            // Touchscreen.
```

Comment:
```suggestion
            // Touch Screen.
```
I'm not sure about this one but I don't think we generally write touch screen as if it was a single word?

Reply:
I think it should be touch screen, but I'm not so sure about the dot at the end.

Reply:
If other places are calling it "Touchscreen", then just keep it as "Touchscreen".